src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs

```diff
@@ -193,18 +194,37 @@ public override void LoadView()
 		public override void ViewWillAppear(bool animated)
 		{
 			base.ViewWillAppear(animated);
-			ConstrainItemsToBounds();
```

Comment:
This makes no sense here because bounds are not set yet: they'll be in `ViewWillLayoutSubviews`.
So any computation made here cannot be trustworthy.

Reply:
If we navigate from a CV page to a new page and then back to the CV sometimes ViewWillLayoutSubviews will not always trigger.

Reply:
Interesting, but why is then necessary to trigger this method again considering the view was already laid out before pushing the new page? @rmarinho